### ISSUE TITLE
Functionality to freeze the top row and/or first column in the DataTable plugin is needed

### ISSUE THREAD

**Issue by ashildskalnes** [equinor/webviz-config#571]
Webviz-4D uses DataTables to display well and production information, but if the table contains many rows and/or columns it becomes difficult to view the content. Freeze options like in Excel would solve this problem. This feature request was initially proposed by one of the partners in the Johan Sverdrup field.

**Comment by anders-kiaer** [comment]
Good idea to freeze first row and/or column for large table. 👍  I'm not sure if this issue belongs in this repository though, since `dash` DataTable is maintained + developed upstream 🤔 

Could it be that it already is suppported? I see `fixed_columns` is mentioned in DataTable documentation. Maybe something similar for freezing top row on scroll? Alternatively utilize the pagination support.
